src/AccountData.js

```diff
@@ -1,68 +1,42 @@
-import { drizzleConnect } from 'drizzle-react'
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
+import React, { Component } from 'react';
 
 /*
  * Create component.
  */
 
 class AccountData extends Component {
-  constructor(props, context) {
-    super(props);
-
-    this.precisionRound = this.precisionRound.bind(this);
-  }
-
-  precisionRound(number, precision) {
-    var factor = Math.pow(10, precision)
-    return Math.round(number * factor) / factor
-  }
-
-  render() {
-    // No accounts found.
-    if(Object.keys(this.props.accounts).length === 0) {
-      return (
-        <span>Initializing...</span>
-      )
-    }
-
-    // Get account address and balance.
-    const address = this.props.accounts[this.props.accountIndex]
-    var balance = this.props.accountBalances[address]
-    const units = this.props.units ? this.props.units.charAt(0).toUpperCase() + this.props.units.slice(1) : 'Wei'
-
-    // Convert to given units.
-    if (this.props.units) {
-      balance = this.context.drizzle.web3.utils.fromWei(balance, this.props.units)
-    }
-
-    // Adjust to given precision.
-    if (this.props.precision) {
```

Comment:
We're lacking the `precision` prop from the old implementation. This should fix the wrong decimals issue.

Reply:
Fixed this one.